src/fheroes2/editor/editor_save_map_window.cpp

```diff
@@ -241,6 +241,7 @@ namespace Editor
         background.applyTextBackgroundShading( mapNameRoi );
         fheroes2::ImageRestorer mapNameBackground( display, mapNameRoi.x, mapNameRoi.y, mapNameRoi.width, mapNameRoi.height );
 
+        mapNameText.fitToOneRow( mapNameRoi.width );
```

Comment:
Hi, @ihhub, to address the issue mentioned by @Branikolog we can reduce the maximum text width here:
```suggestion
        mapNameText.fitToOneRow( mapNameRoi.width - 6 );
```